### ISSUE TITLE
how to finetune a pretrained model with new speakers

### ISSUE THREAD

**Issue by wwyl2000** [TaoRuijie/ECAPA-TDNN#73]
I have a question about the continuous training:
How to finetune the model?

Thanks!

**Comment by TaoRuijie** [comment]
I am not very sure since I have not tried this setting. 

From my opinion, since X >> N, finetuning with N speakers with the additional data only with a smaller learning rate maybe is the best. Fineting with (X+N) speakers with all data only with a smaller learning rate is also a suitable solution.
